src/eval.c

```diff
@@ -3605,6 +3617,293 @@ get_lit_string_tv(char_u **arg, typval_T *rettv, int evaluate)
     return OK;
 }
 
+int get_repeated_single_quotes_length(char_u *x, char_u *whole_start) {
```

Comment:
This can be static :+1:

I fixed this to below

```c
    static int
get_repeated_single_quotes_length(char_u *x, char_u *whole_start)
{
    char_u *follower;
    char_u *following;
    ...
```